patches/net/minecraft/client/renderer/culling/Frustum.java.patch

```diff
@@ -5,7 +5,7 @@
  
      public boolean isVisible(AABB p_113030_) {
 +        // FORGE: exit early for infinite bounds, these would otherwise fail in the intersection test at certain camera angles (GH-9321)
-+        if (p_113030_.equals(net.neoforged.neoforge.client.extensions.IBlockEntityRendererExtension.INFINITE_EXTENT_AABB)) return true;
++        if (p_113030_.equals(AABB.INFINITE)) return true;
```

Comment:
Would it be preferrable to have a `AABB#isInfinite()` method along the lines of
```java
public boolean isInfinite() {
    return this == INFINITE || (this.min[X/Y/Z] == Double.NEGATIVE_INFINITY && this.max[X/Y/Z] == Double.POSITIVE_INFINITY);
}
```
?

Reply:
Sure, I think we found `Double.compare` is slow in some cases anyway.